com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -187,24 +224,34 @@ private static Quaternion ToQuaternion(Vector3 eigenvector0, Vector3 eigenvector
 #endregion
 
 #region Export
-        public Link.Inertial ExportInertialData() 
+        public Link.Inertial ExportInertialData()
         {
-            ArticulationBody robotLink = GetComponent<ArticulationBody>();
+            var robotLink = GetComponent<ArticulationBody>();
 
             if (robotLink == null)
+            {
+                Debug.LogWarning("No data to export.");
                 return null;
+            }
 
             UpdateLinkData();
-            Vector3 originAngles = inertialAxisRotation.eulerAngles;
-            Origin inertialOrigin = new Origin(robotLink.centerOfMass.Unity2Ros().ToRoundedDoubleArray(), new double[] { (double)originAngles.x, (double)originAngles.y, (double)originAngles.z });
-            Link.Inertial.Inertia inertia = ExportInertiaData();
+            return ToLinkInertial(robotLink);
+        }
+
+        Link.Inertial ToLinkInertial(ArticulationBody robotLink)
+        {
+            var originAngles = inertialAxisRotation.eulerAngles;
+            var inertialOrigin = new Origin(
+                robotLink.centerOfMass.Unity2Ros().ToRoundedDoubleArray(), 
+                new double[] { (double)originAngles.x, (double)originAngles.y, (double)originAngles.z });
+            var inertia = ExportInertiaData();
 
-            return new Link.Inertial(Math.Round(robotLink.mass, RoundDigits), inertialOrigin, inertia);
+            return new Link.Inertial(Math.Round(robotLink.mass, k_RoundDigits), inertialOrigin, inertia);
         }
 
         private Link.Inertial.Inertia ExportInertiaData()
```

Comment:
Oh right ExportInertiaData() is right here... Maybe we could disambiguate these two methods a little more since we're doing some changes round here?

Reply:
I think a broader re-factor is way out of scope for this bugfix; I've already spent way more time than I meant to on clean-up in here. I'd recommend doing that as part of a larger renaming initiative when you're confident it's not going to create a slew of merge conflicts.
![image](https://user-images.githubusercontent.com/29758400/163844464-64457060-2924-4e65-893d-2b1629c77a0c.png)
![image](https://user-images.githubusercontent.com/29758400/163844579-da3f20c5-8348-4e4f-8155-cfa8c3dc4d0d.png)
![image](https://user-images.githubusercontent.com/29758400/163844627-c01c3d63-0225-43e2-bb5f-227e906655ef.png)

Reply:
You're right, I guess it's a can of worms that is waiting to be tackled. We already have it in our plans to do a re-work of the URDF importer so that's where we can make these changes.